takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -5,68 +5,65 @@
-              <a class="nav-link{% if not forloop.last %} border-right {% else %} {% endif %}" 
-                href="{{ settings.takwimu.SocialMediaSetting|object_value:social_media }}" target="_blank" rel="noopener">
+            <li class="nav-item">
+              <a class="nav-link float-left" href="{{ settings.takwimu.SocialMediaSetting|object_value:social_media }}"
+                target="_blank" rel="noopener">
                 <i class="fab fa-{{ social_media }} fa-fw"></i>
               </a>
+              {% if not forloop.last %}<div class="border-right m-2 float-right" style="height:24px; width:1px; opacity:0.2;"></div> {% endif %}
             </li>
           {% endif %}
         {% endfor %}
-      </ul>
+        </ul>
     </div>
-  </div>
 
-  <div class="row pt-5">
-    <div class="col-8">
-      <p class="mb-0 text-truncate" style="opacity: 0.6; font-size: 16px">Takwimu is an initiative by:</p>
-      <div class="row">
-        <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ap-white.png' %}" class="img-responsive" alt="africapractice">
-        </a>
-        <a href="https://codeforafrica.org" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
-        </a>
-        <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
-        </a>
+    <div class="row pt-5">
+      <div class="col-12">
+        <p class="mb-0 text-truncate font-weight-bold">An initiative by </p>
+        <div class="row pl-1">
+          <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/ap-footer-white.png' %}" class="img-responsive" alt="africapractice">
+          </a>
+          <a href="https://codeforafrica.org" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
+          </a>
+          <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
+          </a>
+          <div class="col-4">
```

Comment:
With `.row` defined, the corresponding `.col-8` is missing for the 3 logos above this `div.col-4`. Please fix.